nautobot/ipam/models.py

```diff
@@ -386,46 +385,6 @@ def get_utilization(self):
         return UtilizationData(numerator=child_prefixes.size, denominator=self.prefix.size)
 
 
-@extras_features(
-    "custom_fields",
-    "custom_validators",
-    "graphql",
-    "relationships",
-)
-class Role(OrganizationalModel):
-    """
-    A Role represents the functional role of a Prefix or VLAN; for example, "Customer," "Infrastructure," or
-    "Management."
-    """
-
-    name = models.CharField(max_length=100, unique=True)
-    slug = AutoSlugField(populate_from="name")
-    weight = models.PositiveSmallIntegerField(default=1000)
```

Comment:
Does changing from a `ipam.Role.weight` default of `1000` to a `extras.Role.weight` default of `None` have any impacts to the Prefix and VLAN models? I don't think there are a lot of places we actually **use** `weight` in the core code, but want to be sure.

Reply:
On data that currently exists No I dont think there is any impact, because all records would be transferred. But when creating a new Role with its '`content type`' set to '`ipam.Vlan`'/ '`ipam.Prefix`', That default of 1000 would not be automatically added, I suppose that would be the impact

Reply:
Understood. I was asking more specifically what impact that has on the operation of the application - i.e. is there anywhere that's expecting all Prefix/VLAN roles to have a non-null `weight` that would have issues as a result of this change?

Reply:
I dont think so/ I cant find any